integration-tests/camel/camel-jdbc/pom.xml

```diff
@@ -77,6 +77,22 @@
     </build>
 
     <profiles>
+        <profile>
+            <!--
+                Disable tests for Quarkus SNAPSHOTs since camel-quarkus-jdbc 1.1.0 is not compatible.
+                This can be removed when camel-quarkus supports quarkus >= 1.9.x.
```

Comment:
Note that the new datasource configuration is there since 1.3 so you can already migrate to it.

Obviously, it's not worth a release but I thought I should mention it.